lstchain/reco/r0_to_dl1.py

```diff
@@ -507,11 +515,6 @@ def r0_to_dl1(
             # write sub tables
             if is_simu:
                 write_subarray_tables(writer, event, metadata)
-                cal_mc(event)
```

Comment:
Why was this here? Isn't the calibration done by r1_dl1_calibrator below, both for data and mc?

Reply:
Small pixel charges change a bit (even when no additional random NSB is added) - not clear if this cal_mc had some role...

Reply:
No, there seems to be some random component in the signal extraction, even running twice the original version results in slightly different pixel charges

Reply:
(deleted comments about tests were wrong - I had not really switched off the additional NSB generation)